.github/workflows/docker_publish.yml

```diff
@@ -44,6 +44,14 @@ jobs:
     steps:
       - name: Checkout repository
         uses: actions/checkout@v2
+      
+      - name: Environment setup
+        run: |
+          if [ "${{ matrix.isMerged }}" == "true" ]; then
```

Comment:
Maybe I don't understand how the `isMerged` works, but this seems like the logic is reversed.  Don't we want the sha for PRs and without a sha for merged?